tests/roxctl/bats-tests/local/deployment-bundles-psps.bats

```diff
@@ -23,7 +23,7 @@ central_bundle_psp_enabled() {
     shift
     local bundle_dir="${out_dir}/bundle-central-${RANDOM}-${RANDOM}-${RANDOM}"
     roxctl-release central generate "${cluster_type}" pvc "$@" --output-dir="${bundle_dir}"
-    run grep -rq "kind: PodSecurityPolicy" "${bundle_dir}"
+    run grep -rq "kind: PodSecurityPolicy" "${bundle_dir}/central"
```

Comment:
This was a bug in the test suite. Before it was not only grepping the deployment bundle, but also a copy of the Helm chart, which is included in that output directory as well for convenience reasons.